spel/minimal-linux.json

```diff
@@ -345,23 +458,31 @@
         "azure_virtual_network_name": "",
         "azure_vm_size": "Standard_DS2_v2",
         "iso_url_centos7": "http://mirror.cs.vt.edu/pub/CentOS/7/isos/x86_64/CentOS-7-x86_64-Minimal-1908.iso",
+        "iso_url_centos8": "http://mirror.cs.vt.edu/pub/CentOS/8/isos/x86_64/CentOS-8.1.1911-x86_64-dvd1.iso",
         "root_volume_size": "20",
         "security_group_cidrs": "0.0.0.0/0",
         "source_ami_centos7_hvm": "ami-090b9dabe1c9f40b3",
+        "source_ami_centos8_hvm": "ami-0c5f24fae8f05460e",
         "source_ami_rhel7_hvm": "ami-0394fe9914b475c53",
+        "source_ami_rhel8_hvm": "ami-0c322300a1dd5dc79",
         "spel_amigen7branch": "master",
         "spel_amigen7manfst": "",
         "spel_amigen7pkggrp": "core",
         "spel_amigen7source": "https://github.com/plus3it/AMIgen7.git",
         "spel_amigen7storlay": "/:rootVol:4,swap:swapVol:2,/home:homeVol:1,/var:varVol:2,/var/log:logVol:2,/var/log/audit:auditVol:100%FREE",
+        "spel_amigen8bootsize": "20",
+        "spel_amigen8branch": "master",
+        "spel_amigen8manfst": "",
+        "spel_amigen8source": "https://github.com/plus3it/AMIgen8.git",
+        "spel_amigen8storlay": "",
```

Comment:
Freaking GitHub highlighting three lines... You're referring to the storage-layout option? If so, then "already supported".

- [DiskSetup.sh](https://github.com/plus3it/AMIgen8/blob/master/DiskSetup.sh#L79-L179)
- [MkChrootTree.sh](https://github.com/plus3it/AMIgen8/blob/master/MkChrootTree.sh#L111-L155)
- And relevant pass-throughs in spel's `amigen8-build.sh` wrapper-scripts

Reply:
right, referring to the line the comment is actually on, `spel_amigen8storlay`. my confusion is that for the `spel_amigen7storlay` just above, we had to set the default value to something actually valid. is the implementation just that much different? and can we adjust so these work the same across both builds?

Reply:
The AMIGen8 scripts (and the `amigen8-build.sh` wrapper) are set up to do nothing if the value is null. Can refactor the AMIgen7 (most likely the wrapper) per the #395. Still plan to keep the storage-layout vars separate in case we ultimately need to implement different layouts for EL7 and EL8